Week1/cash.c

```diff
@@ -0,0 +1,25 @@
+#include <cs50.h>
+#include <stdio.h>
+
+void cashback();
+
+int main(void){
+    float owe = get_float("Change owed: ");
+    int penny = (int)(owe*100);
+    if (penny>0){
+        cashback(penny);
+    }else{
+        printf("foo\n");
+    }
+}
+
+void cashback(int p){
```

Comment:
Bad names variables:

```suggestion
void cashback(int penny_value){
```

what is `p` meant to be?